apps/src/templates/sectionProgressV2/ProgressIcon.jsx

```diff
@@ -38,7 +38,7 @@ export default function ProgressIcon({itemType}) {
   );
 
   return (
-    <>
+    <div data-testid="progress-icon">
```

Comment:
I tried adding this `testid` to the `React.Fragment` but apparently `React.Fragments` can only have a `key` component.  Thus I changed this to a `div`.  It did not seem to impact rendering at all.  Open to other suggestions here though.  It is possible that we remove this and just spot check that one of the `ITEM_TYPES` gets rendered specifically.